webservices/partition/sched_a.py

```diff
@@ -40,6 +42,7 @@ def column_factory(cls, parent):
                 parent.c.contbr_id,
                 parent.c.cmte_id,
             ).label('clean_contbr_id'),
+            sa.cast(sa.func.get_transaction_year(parent.c[cls.transaction_date_column], parent.c.rpt_yr), sa.Numeric(4, 0)).label('transaction_two_year_period'),
```

Comment:
The result of overthinking without seeing the simple solution and attempting to match other column types, e.g., `rpt_yr`.  A change to `smallint` works just fine; updates coming soon.  I'm addressing the flake8 warnings as well.